sotodlib/io/imprinter.py

```diff
@@ -1541,8 +1541,8 @@ def get_readout_ids_for_book(self, book):
                     f"Readout IDs not found for {obs_id}. Indicates issue with G3tSmurf Indexing"
                 )
             if np.any(checks):
-                self.logger.warning(
-                    f"Found {sum(checks)} channels without readout_id. Were fixed tones running?"
+                self.logger.info(
+                    f"Found {sum(checks)} channels without readout_id. Assume fixed tones are running."
```

Comment:
Is there no way to check that fixed tones are running other than finding a channel without a readout_id?

Reply:
As far as I know we don't actually have fixed tones in the .g3 file headers in any way. So that's what we've had to do. If it was in the headers then we could be more careful